include/IntersonArrayCxxImagingContainer.h

```diff
@@ -118,6 +118,8 @@ class IntersonArrayCxx_EXPORT Container
 
   void SetHWControls(IntersonArrayCxx::Controls::HWControls * controls);
 
```

Comment:
A comment here describing what this function does and when it is used would be helpful.